codespan-reporting/src/term/views.rs

```diff
@@ -346,9 +345,10 @@ where
                             // To render the line right, we have to get back the original labels.
                             let labels = labeled_file
                                 .lines
-                                .iter()
-                                .find(|(index, _)| **index == line_index + 1)
-                                .map_or(&current_labels, |(_, line)| &line.multi_labels);
+                                .get(&(line_index + 1))
+                                .map_or(&[] as &[(usize, LabelStyle, MultiLabel)], |line| {
```

Comment:
I don't know if I did something wrong, but Rust would not accept just `&[]` and would error on the map function with: `expected array of 0 elements, found slice`.

Reply:
Ahh yeah that is annoying - I think I've done `&[][..]` in the past to fix this